.github/workflows/boostrap-workflow.yml

```diff
@@ -0,0 +1,23 @@
+name: bootstrap workflow
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 2
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+    - name: run boostrap.sh
+      run: |
+        cd ./
+        sh bootstrap.sh
```

Comment:
Change this line to:
./bootstrap.sh
The dot tells the shell that it doesn't need to search for the executable in other directories mentioned in the PATH environment variable. The shell resolves which executable to run by taking it from current directory.